js/handlers/button.js

```diff
@@ -41,48 +41,47 @@ define([
             // inject the button at post render
             if (!this.isTrickleEnabled(view.model)) return;
 
-            view.$el.append(this.buttonViews[view.model.get("_id")].$el);
+            view.$el.append(this.buttonViews[view.model.get('_id')].$el);
         },
 
         isTrickleEnabled: function(model) {
             var trickle = Adapt.trickle.getModelConfig(model);
             if (!trickle || !trickle._isEnabled) return false;
 
-            if (trickle._onChildren && model.get("_type") === "article") return false;
+            if (trickle._onChildren && model.get('_type') === 'article') return false;
 
             return true;
         },
 
         setupConfigDefaults: function(model) {
-            if (model.get("_isTrickleButtonConfigured")) return;
+            if (model.get('_isTrickleButtonConfigured')) return;
+
+            var defaults = {
+                _isEnabled: true,
+                _styleBeforeCompletion: 'hidden',
+                _styleAfterClick: 'hidden',
+                _isFullWidth: true,
+                _autoHide: false,
+                _className: '',
+                text: 'Continue',
+                startText: 'Begin',
+                finalText: 'Finish',
+                _component: 'trickle-button',
+                _isLocking: true,
+                _isVisible: false,
+                _isDisabled: false
+            };
 
             var trickle = Adapt.trickle.getModelConfig(model);
-            trickle._button = _.extend({
-                "_isEnabled": true, //(default=true)
-                "_styleBeforeCompletion": "hidden", //(default=hidden)
-                "_styleAfterClick": "hidden", //(default=hidden)
-                "_isFullWidth": true, //(default=true)
-                "_autoHide": true, //(default=true)
-                "_className": "", //(default="")
-                "text": "Continue", //(default="Continue")
-                "startText": "Begin", //(default="Begin")
-                "finalText": "Finish", //(default="Finish")
-                "_component": "trickle-button", //(default="trickle-button")
-                "_isLocking": true,
-                "_isVisible": false,
-                "_isDisabled": false
-            }, trickle._button);
-
+            trickle._button = _.extend(defaults, trickle._button);
 
             if (trickle._button._isFullWidth) {
                 trickle._stepLocking._isEnabled = true;
-                trickle._button._styleAfterClick = "hidden";
-            } else {
-                trickle._button._autoHide = false;
```

Comment:
Why did you remove this line?

Reply:
because I didn't think it through properly ;-) I'll revert that change shortly.

Reply:
Cool :)

Reply:
all done, thanks for catching that one!